src/common-components/RedirectLogistration.jsx

```diff
@@ -36,6 +37,10 @@ const RedirectLogistration = (props) => {
       // TODO: Do we still need this cookie?
       setCookie('van-504-returning-user', true);
       const registrationResult = { redirectUrl: finalRedirectUrl, success };
+      if (isRegistrationEmbedded) {
+        window.parent.postMessage({ message: 'redirection', redirectUrl: `${window.location.origin}${AUTHN_PROGRESSIVE_PROFILING}` }, '*');
```

Comment:
you need to specify the `targetOrigin` instead of `*` which is `https://discover.edx.org`.

Reply:
if the current structure is not a final decision, can we restructure the message object like this?
```
{ action: 'redirect` }
```
thoughts?

Reply:
can we move the `redirection` str to constants?